### ISSUE TITLE
Выбор очерёдности платежа в платёжных документах банка по умолчанию

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2060]
Originally reported on Google Code with ID 2060

```
В данный момент подставляется напрямую из скриптов BN_Payment_SyncField и BN_Payment_SetupRecord
```

Reported by `KlimenkoNM` on 2010-06-22 10:59:09

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-06-22 13:35:21
- **Labels added**: Bank
- **Labels removed**: BANK

**Comment by gsbelarus** [comment]
```
Issue 1882 has been merged into this issue.
```

Reported by `alexandra.gsoftware` on 2011-02-25 18:37:53

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2011-02-25 18:39:07
- **Status changed**: `Duplicate`
- **Merged into**: #2156
